index.bs

```diff
@@ -12647,7 +12646,8 @@ internal method as follows.
     1.  Let |prototype| be |O|.\[[GetPrototypeOf]]().
     1.  While |prototype| is not null:
         1.  If |prototype| is not a [=named properties object=],
-            and |prototype| has an own property named |P|, then return false.
+            and <a abstract-op>HasOwnProperty</a>(|prototype|, |P|) is <emu-val>true</emu-val>,
```

Comment:
Is the potentially-observable change here what happens when there's a proxy on the proto chain?  What do browsers currently do here?

Reply:
They're inconsistent, of course. I'll retake this part after #607 is fixed.